### PR TITLE
Update mcli yamls

### PR DESCRIPTION
**Pull request by dakinggg**
Updates the yamls to the latest release.

### CODE REVIEW

File: mcli/mcli-convert-composer-to-hf.yaml [hunk 1]
```diff
@@ -13,7 +13,7 @@ command: |
     --hf_output_path s3://bucket/folder/hf/ \
     --output_precision bf16 \
 
-image: mosaicml/llm-foundry:2.3.1_cu121-latest
+image: mosaicml/llm-foundry:2.4.0_cu124-latest
```
Comment (mvpatel2000) [inline mcli/mcli-convert-composer-to-hf.yaml:16]:
```suggestion
image: mosaicml/llm-foundry:2.4.1_cu124-latest
```

**Review by mvpatel2000 — COMMENTED** [review]
should be 2.4.1

File: mcli/mcli-convert-composer-to-hf.yaml (hunk 1 above)
Comment (snarayan21) [reply mcli/mcli-convert-composer-to-hf.yaml:16]: we don't support torch 2.4.1 in foundry yet, can go in a separate PR

**Review by snarayan21 — APPROVED** [review]
lgtm

**Pull request merged**